Tests fail because the server banner contains "Gentoo" instead of "Unix". Disable them.


diff -ruN mod_perl-2.0.10-rc1.orig/t/response/TestAPI/server_const.pm mod_perl-2.0.10-rc1/t/response/TestAPI/server_const.pm
--- mod_perl-2.0.10-rc1.orig/t/response/TestAPI/server_const.pm	2016-09-30 14:45:31.000000000 +0200
+++ mod_perl-2.0.10-rc1/t/response/TestAPI/server_const.pm	2016-10-01 13:18:02.359794404 +0200
@@ -25,7 +25,7 @@
 
     my $r = shift;
 
-    plan $r, tests => 6;
+    plan $r, tests => 4;
 
     # test Apache2::ServerUtil constant subroutines
 
@@ -45,13 +45,13 @@
     ok t_cmp($server_descr, qr!\bworld domination series/2\.0\b!,
              'Apache2::ServerUtil::get_server_description() -- component');
 
-    # assuming ServerTokens Full (default) the banner equals description
-    ok t_cmp(Apache2::ServerUtil::get_server_banner, $server_descr,
-             'Apache2::ServerUtil::get_server_banner()');
-
-    # version is just an alias for banner
-    ok t_cmp(Apache2::ServerUtil::get_server_version, $server_descr,
-             'Apache2::ServerUtil::get_server_version()');
+#    # assuming ServerTokens Full (default) the banner equals description
+#    ok t_cmp(Apache2::ServerUtil::get_server_banner, $server_descr,
+#             'Apache2::ServerUtil::get_server_banner()');
+
+#    # version is just an alias for banner
+#    ok t_cmp(Apache2::ServerUtil::get_server_version, $server_descr,
+#             'Apache2::ServerUtil::get_server_version()');
 
     Apache2::Const::OK;
 }
